mods/Fifty.ServerUtilities/mod/scripts/vscripts/fsu_core.nut

```diff
@@ -117,35 +121,87 @@ void function FSU_PrivateChatMessage( entity player, string message ) {
  * Returns the maximum number of pages for FSU_PrintFormattedList
  * @param list The list to be printed
  * @param columns The nubre of columns per row
+ * @param separator The list item separator, inconsquential as long as charachter count is the same
 */
-int function FSU_GetListPages( array< string > list, int columns = 1 ) {
-	return int( ceil( list.len() / ( columns * 5.0 ) ) )
+int function FSU_GetListPages( array< string > list, int rows = 5, string separator = "%T, " ) {
+	string row = ""
+	array <string> rowList
+	foreach(string item in list){
+		if( row == ""){
+			row = "  " + item
+		}
+		else{
+			row += separator + "  " + item
+		}
+
+		if( row.len() > 75){
+			rowList.append(row)
+			row = ""
+		}
+	}
+	rowList.append(row)
+
+	return int( ceil( rowList.len() / ( rows * 1.0 ) ) ) // For some reason this does not use floats without the * 1.0 and as such would sometimes return zero when it shouldn't

```

Comment:
You can use ` float( int )` to convert int to float. As `rows` and `rowList.len()` are ints the return type is int which is why it'd sometimes return 0.

Reply:
Will do. This issue was there before, it would cause there to be "0" pages if you tried to make them too dense and I tracked it down to this.